charter/index.html

```diff
@@ -269,9 +269,12 @@ <h3>
         <section id="dependencies">
           <h3>Dependencies</h3>
 
-          <p>Depending on the W3C Solid Community Group progress, including consideration for <a href="https://www.w3.org/2021/Process-20211102/#adequate-implementation">adequate implementation experience</a>, the Group will also produce W3C Recommendations for the following documents:</p>
+          <p>Depending on the Group progress, including consideration for <a href="https://www.w3.org/2021/Process-20211102/#adequate-implementation">adequate implementation experience</a>, the Group will also produce W3C Recommendations for the following documents:</p>
```

Comment:
If by *the Group* you mean proposed Solid WG I think it should just directly refer to it.